ui/app/helpers/is-wildcard-string.js

```diff
@@ -6,7 +6,11 @@ export function isWildcardString([string]) {
   }
   // string is actually an object which is what comes in in the searchSelect component
   if (typeof string === 'object') {
-    string = Object.values(string)[0];
+    // if the dropdown is used the object is a class which cannot be converted into a string
```

Comment:
We may want to also mention that we know it is not a wilcard, since being a model means it is a single result? Up to you. Maybe this comment on the PR will suffice :D

Reply:
Good point.  We'll keep this comment as a reminder.